docs/tutorials/detection/demo_webcam.py

```diff
@@ -0,0 +1,118 @@
+"""9. Run an object detection model on your webcam
+==================================================
+
+This article will shows how to play with pre-trained object detection models by running
+them directly on your webcam video stream.
+
+.. note::
+
+    - This tutorial has only been tested in a MacOS environment
+    - Python packages required: cv2, matplotlib
+    - You need a webcam :)
+    - Python compatible with matplotlib rendering, installed as a framework in MacOS see guide `here <https://matplotlib.org/faq/osx_framework.html>`__
+
+
+Loading the model and webcam
+----------------------------
+Finished preparation? Let's get started!
+First, import the necessary libraries into python.
+
+.. code-block:: python
+
+    import time
+
+    import cv2
+    import gluoncv as gcv
+    import matplotlib.pyplot as plt
+    import mxnet as mx
+
+
+In this tutorial we use ``ssd_512_mobilenet1.0_voc``, a snappy network with good accuracy that should be
+well above 1 frame per second on most laptops. Feel free to try a different model from
+the :doc:`../../model_zoo/index` !
```

Comment:
:doc:`..` does not get rendered properly. Try 
```
`Gluon Model Zoo <../../model_zoo/index.html>`__
```

Reply:
updated

Reply:
Sorry, my bad, it's  ``../../model_zoo/detection`` actually

Reply:
hard to get right :)